appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: 1.6.{build}
```

Comment:
Is there a reason this is 1.6? I usually just set it to `{build}` to get an incrementing number like Travis

Reply:
I did that to align it with the version number in the `psd1` file

Reply:
okay, I'll revert that on #30 then